python/tvm/relay/frontend/tflite.py

```diff
@@ -704,6 +705,35 @@ def convert_conv(self, op, conv_type):
 
         return out
 
+    def convert_split(self, op):
+        """split implementation."""
+        try:
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.Operator import Operator
+            from tflite.SplitOptions import SplitOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+
+        assert len(input_tensors) == 2, "input tensors length should be == 2"
+
+        axis_tensor = input_tensors[0]
+        split_axis = self.get_tensor_value(axis_tensor)
+        input_tensor = input_tensors[1]
+        input_tensor_idx = input_tensor.tensor_idx
+
+        assert op.BuiltinOptionsType() == BuiltinOptions.SplitOptions
+        op_options = op.BuiltinOptions()
+        split_options = SplitOptions()
+        split_options.Init(op_options.Bytes, op_options.Pos)
+        num_splits = split_options.NumSplits()
+
+        in_expr = self.get_expr(input_tensor_idx)
+        out = _op.split(in_expr, num_splits, axis=int(split_axis))
```

Comment:
Add code:
```python
if len(out) == 1:
  out = out[0]
```
Because Relay assume `split` op always return TupleWrapper, when split 1 we return just one output. Otherwise, we `set_expr` will get wrong type (TupleWrapper).

Reply:
I see your point but len won't cut it as we would always get a TupleWrapper back and the length is 1,  but what we need to do is to interrogate its size and that seems to work.

Reply:
Testing a fix now.